plugins/leemons-plugin-learning-paths/frontend/src/components/ModuleDashboard/helpers/previewHooks.js

```diff
@@ -8,6 +8,21 @@ import prepareAsset from '@leebrary/helpers/prepareAsset';
 import { capitalize, get, keyBy, map, mapValues } from 'lodash';
 import { useEffect, useMemo } from 'react';
 
+function useSubjectsData(module) {
+  const { assignable } = module || {};
+  const subjects = map(assignable?.subjects, 'subject') ?? [];
```

Comment:
It should be inside of useMemo, to avoid processing.

Reply:
And should come from `useClassesSubjects(activity?.classes)` not assignable

Reply:
Done!